packages/pluggableWidgets/app-events-native/src/AppEvents.tsx

```diff
@@ -95,8 +100,109 @@ export class AppEvents extends Component<Props> {
 
         this.isConnected = nextState.isConnected;
     }
+
+    private triggerOnUnloadEvent(): void {
+        if (this.props.onUnloadAction && this.props.onUnloadAction.canExecute) {
```

Comment:
`this.props.onUnloadAction?.canExecute` ?

Reply:
good point